.gitignore

```diff
@@ -0,0 +1 @@
+schema_doc*
```

Comment:
This `.gitignore` doesn't apply to the publishing workflow, but it's here such that when building locally one doesn't accidentally commit the generated files.

Reply:
I see. In that case should the `.gitignore` not exclude `*.html` as well? Though I guess this could be too broad, so I'd be fine with either way.

Reply:
Ah, I now see your point: the precise command from the workflow generates `pages/index.html`. When working locally, I just used
```console
generate-schema-doc --config-file jsfh.yml witness.schema.json
```
which generates `schema_doc.html` into the root directory.

I'll add the `pages` directory also to `.gitignore` so it's covered either way.

Reply:
Alright, this is probably the best we can do here.

The reason I didn't suggest this is because one could still run, e.g.
```
generate-schema-doc --config-file jsfh.yml witness.schema.json random_page_title.html
```